### PR TITLE
ELPP-2362 Updated Lax fragment integration.

### PR DESCRIPTION
**Pull request by jamiehollern**
@nlisgo

### CODE REVIEW

File: src/modules/jcms_article_fragment/jcms_article_fragment.install [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+/**
+ * @file
+ * Contains jcms_article_fragment install/update functions.
+ */
+
+/**
+ * Implements hook_update_N().
+ */
+function jcms_article_fragment_update_8001() {
+  \Drupal::entityDefinitionUpdateManager()->applyUpdates();
```
Comment (nlisgo) [inline src/modules/jcms_article_fragment/jcms_article_fragment.install]: Why is this necessary?

Comment (jamiehollern) [inline src/modules/jcms_article_fragment/jcms_article_fragment.install]: The article fragment entity schema has changed in this piece of work from the original piece of work. This simply updates it in places where it's already installed.

Comment (nlisgo) [inline src/modules/jcms_article_fragment/jcms_article_fragment.install]: This should be handled with:

```
drush cim -y
```

which should be run as a deploy step in all environments. While it does no harm to add it, it should not be necessary so please remove.

File: src/modules/jcms_article_fragment/jcms_article_fragment.install [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+/**
+ * @file
+ * Contains jcms_article_fragment install/update functions.
+ */
+
+/**
+ * Implements hook_update_N().
+ */
+function jcms_article_fragment_update_8001() {
```
Comment (nlisgo) [inline src/modules/jcms_article_fragment/jcms_article_fragment.install]: Nitpick: should be `8101` I believe.

File: src/modules/jcms_article_fragment/jcms_article_fragment.install [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ * @file
+ * Contains jcms_article_fragment install/update functions.
+ */
+
+/**
+ * Implements hook_update_N().
+ */
+function jcms_article_fragment_update_8101() {
+  \Drupal::entityDefinitionUpdateManager()->applyUpdates();
+}
```
Comment (jamiehollern) [inline src/modules/jcms_article_fragment/jcms_article_fragment.install]: @nlisgo Having double checked this, this hook is necessary. The schema is not stored in configuration but in code so it needs to be updated this way.

File: src/modules/jcms_article_fragment/jcms_article_fragment.install (hunk 2 above)
Comment (jamiehollern) [inline src/modules/jcms_article_fragment/jcms_article_fragment.install]: Done.

**Comment by nlisgo** [comment]
I'll merge as soon as travis goes green!
